conf/messages

```diff
@@ -1444,6 +1444,26 @@ incomeSources.ceaseForeignProperty.checkDetails.dateStopped           = Date bus
 incomeSources.ceaseForeignProperty.checkDetails.confirm               = Confirm and continue
 incomeSources.ceaseForeignProperty.checkDetails.change                = Change
 
+## Cease - Check Answers ##
+cease-check-answers.title                                     = Check your answers
+cease-check-answers.heading                                   = Sole trader
+cease-check-answers.cease-date                                = Date business stopped
+cease-check-answers.business-name                             = Business name
+cease-check-answers.trade                                     = Business trade
+cease-check-answers.address                                   = Address
+cease-check-answers.change                                    = Change
+cease-check-answers.unknown                                   = Unknown
```

Comment:
❓ could use general Unknown message as if one changes it's likely that they will all change